lib/WorkflowMetabenchmarks.groovy

```diff
@@ -1,19 +1,15 @@
 //
-// This file holds several functions specific to the workflow/benchmark.nf in the nf-core/benchmark pipeline
+// This file holds several functions specific to the workflow/socialgene.nf in the nf-core/socialgene pipeline
```

Comment:
TODO: Replace mentions of `Socialgene` with `metaBenchmarks`

Reply:
https://github.com/KwanLab/metaBenchmarks/pull/11